homeassistant/components/lm75/.translations/en.json

```diff
@@ -0,0 +1,15 @@
+{
+    "config": {
+        "abort": {
+            "no_devices_found": "No LM75 devices found on the network.",
+            "single_instance_allowed": "Only a single configuration of LM75 is possible."
+        },
+        "step": {
+            "confirm": {
+                "description": "Do you want to set up LM75?",
+                "title": "LM75"
+            }
+        },
+        "title": "LM75"
```

Comment:
The title has moved to the root level in the translation strings.

We've also renamed the translation folder from `.translations` to `translations`. Please rebase on latest dev branch and adjust the translations.